gtfs-realtime-validator-lib/src/test/java/edu/usf/cutr/gtfsrtvalidator/lib/test/UtilTest.java

```diff
@@ -451,10 +451,6 @@ public void testValidPosition() {
         /**
          * Valid lat/longs
          */
-        positionBuilder.setLatitude(0);
-        positionBuilder.setLongitude(0);
-
-        assertEquals(true, GtfsUtils.isPositionValid(positionBuilder.build()));
```

Comment:
I moved this test in the invalid test cases because it seems like 0,0 should be invalid according to #77 and this test broke when I changed the validator.